bin/amphtml-update.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Here is a draft the bash script to update AMP HTML. This script is meant to run in Linux environment (ex. VVV). To run the script, simply `cd` to the plugin root and run `bash bin/amphtml-update.sh`.